app/services/stockit/item_sync.rb

```diff
@@ -71,6 +79,20 @@ def delete
       end
     end
 
+    def dispatch
```

Comment:
yes! I noticed these methods were missing yesterday and pleased to see an endpoint for this. On the Stockit side, it should be calling `item.dispatch` which will handle all the dispatch logic internally.